int32 object_id
string object_identity
geometry_msgs/Pose pose

